dev/WebView2/WebView2AutomationPeer.cpp

```diff
@@ -42,17 +42,27 @@ winrt::Rect WebView2AutomationPeer::GetBoundingRectangleCore()
     return boundingRect;
 }
 
-#if WINUI3
 HRESULT WebView2AutomationPeer::GetRawElementProviderSimple(_Outptr_opt_ IRawElementProviderSimple** value)
 {
+    if (value == nullptr) return S_OK;
```

Comment:
>  if (value == nullptr) return S_OK; [](http://example.com/codeflow?start=2&length=36)

Add null check to _Outptr_opt_ arg to avoid static analysis warning (treated as error) C6011: Dereferencing NULL pointer 'value'.